server/src/controllers/Events/resolver.ts

```diff
@@ -513,8 +513,10 @@ export class EventResolver {
     const updatedEventUser = await prisma.event_users.update({
       data: {
         rsvp: { connect: { name: 'no' } },
-        subscribed: false,
-        ...(eventUser.event_reminder && { event_reminder: { delete: true } }),
+        ...(eventUser.subscribed && {
+          event_reminder: { delete: true },
+          subscribed: false,
+        }),
```

Comment:
This will error if there isn't reminder. Ie. when reminder was already send and deleted.

Reply:
Ya, I thought the subscription is tied to eventReminder, but seeing the code I am wrong

```ts
  @Mutation(() => User)
  async toggleAutoSubscribe(@Ctx() ctx: Required<ResolverCtx>): Promise<User> {
    return await prisma.users.update({
      data: { auto_subscribe: !ctx.user.auto_subscribe },
      where: { id: ctx.user.id },
    });
  }
```

Reply:
I am moving the discussion to the issue, because I am wondering what is the best course going forward

Reply:
There are other emails dependent on the event user `subscribed` status. `subscribed` cannot be used in place of event reminder, as there wouldn't be way to ensure just single reminder is send, flipping `subscribed` after sending reminder would stop also other emails.